integrations/sonarqube/.port/resources/port-app-config.yaml

```diff
@@ -61,6 +61,8 @@ resources:
   - kind: issues
     selector:
       query: 'true'
+      apiQueryParams:
+        resolved: 'false'
```

Comment:
can u also allow `false` & `true` without `'` in the values
i think itll be dummy proof

Reply:
This will not work since Pydantic will convert it to a Python bool (True or False), and this is not one of the allowed values